encryption_works.md

```diff
@@ -96,6 +96,77 @@ It's important to remember that just because you use free software doesn't mean
+- Journalists are concerned that the government now has access to "a record of everywhere I’ve walked, everywhere I’ve been." 
+
+To that end, Freedom of the Press Foundation has supported a variety of encryption tools for journalists that aim at protecting these digital communications. We hope to train journalism organizations in how to use these tools, so they can keep their sources, and themselves, safer.
+
+Once it's clear that journalists have to protect themselves from a variety of threats, not only three-letter government agencies, the next thing to consider is what steps to take to protect themselves from other threats. This line of thinking is called "operational security," or "opsec" for short.
+
+### Operational Security
+
+The NSA isn't the only adversity journalists have to be wary of. Imagine a newsroom that relies on a cloud-based storage system for documents. You and your colleagues are working on a story over the course of a few weeks. Just as you're beginning to edit the final draft, a reporter is fired. He returns his work laptop but nobody thinks to revoke his access to the cloud storage system your newsroom uses. He takes the story and the months of research to a rival newspaper and scoops you.
+
+What steps could have been taken to stop this from happening? Working through this scenario will go a long way towards threat modeling, meaning we figure out the threats facing us and decide how best to defend against them. This starts by asking questions such as:
+
+- What are you protecting?
+- Who/what are you protecting against? 
+- What steps can you take?
+
+Take the above example of the newsroom which got scooped. They want to protect their story -- they don't want all their research to go unrewarded and some other news outlet to get the credit. They're protecting against disgruntled ex-employees as well as external attackers. To prevent this from happening, the newsroom can take steps and implement strict practices where journalists have their access revoked to editorial documents as soon as their employment is terminated.
+
+As you can see, threat modeling is about how you think about a problem as much as what you do to solve it. In this example, we weren't thinking about the NSA or of being hacked by an external adversary. In fact, hacking isn't the biggest danger facing most journalists -- it probably isn't how people will come by your data or compromise you. If you're a journalist, you're much more at risk of your story being scooped, or of falling victim to a phishing attack and someone getting access to all your e-mail. 
+
+Journalists' e-mail accounts are often the subject of attacks, but it's more common for hackers to try and trick Gmail into resetting the password rather than hacking Google itself. There's even a phrase for this -- *rubber-hose cryptanalysis* refers to the fact that it's far easier to coerce or trick someone into giving you their password than break the underlying encryption.
```

Comment:
This isn't the correct use of the term *rubber-hose cryptanalysis*, which refers to the use of physical coercion such as torture to learn secret material such as passwords or encryption keys in order to circumvent otherwise strong encryption.